source/predicates/array.ts

```diff
@@ -158,4 +160,25 @@ export class ArrayPredicate<T = unknown> extends Predicate<T[]> {
 			}
 		});
 	}
+
+	/**
+	Test all elements in the array to match the exact shape of the provided predicate.
+
+	The predicates is an array which describes how the tested array should look like. Each predicate placed at the nth index should be applied to the item at the same index.
```

Comment:
This could still be written more succinctly.

Reply:
what about this?
```

	/**
	Test if the nth element in the array exactly matches with the element placed at the same index in the predictions.
	The predicates is an array which describes how the tested array should look like.

	@param predicates - Predicates to test the array against.


```

Reply:
Yes, sounds better.

Reply:
i have updated the docs, and synced with repo.